src/main/java/io/kestra/plugin/azure/AbstractConnectionInterface.java

```diff
@@ -7,8 +7,7 @@
 
 public interface AbstractConnectionInterface {
     @Schema(
-        title = "The blob service endpoint.",
-        description = "additionally parses it for information (SAS token)"
```

Comment:
Why did you remove the description, was it wrong ?

Reply:
The description is unclear. Does this mean that the endpoint gets parsed to fetch the SAS token from it?

Reply:
Didn't find anything similar in the code, let's merge it like this